src/System Application/App/Feature Key/src/FeatureManagementImpl.Codeunit.al

```diff
@@ -560,4 +560,17 @@ codeunit 2610 "Feature Management Impl."
             RenameFeatureDataUpdateStatus.Rename(RenameFeatureDataUpdateStatus."Feature Key", Rec.Name);
         until FeatureDataUpdateStatus.Next() = 0;
     end;
-}
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Feature Management Triggers", OpenFeatureManagement, '', false, false)]
+    local procedure DefaultOpenFeatureManagement()
+    var
+        FeatureManagementFacade: Codeunit "Feature Management Facade";
+        IsHandled: Boolean;
+        FeatureManagementPageID: Integer;
+    begin
+        FeatureManagementPageID := Page::"Feature Management";
+        FeatureManagementFacade.OnBeforeOpenFeatureManagement(FeatureManagementPageID, IsHandled);
+        if not IsHandled then
```

Comment:
If they don't handle it, we could end up opening a page they provided? 
We should do the following instead: 
1. They handle and open their own page
2. If they do not handle, we open our default page

Reply:
@darjoo so you are just suggesting that I remove FeatureManagementPageID as an arg to OnBeforeOpenFeatureManagement ?

Reply:
It's fine to provide it, but pointless. So yes, I'd suggest to remove it.

Reply:
@darjoo just FYI, I followed the same logic as `"Advanced Settings Ext."::OnBeforeOpenCompanySettings()` in BaseApp. In my opinion it seems like the right thing to do to give in the PageID in case they want to do something, put IsHandled := true and then open the page. It wouldn't make a difference right now, but if we add more logic than opening the page if it's not handled it might make sense in the future. What do you think about that?

Reply:
Why not 
```
if not IsHandled then
    Page.Run(Page::"Feature Management")
else
    Page.Run(FeatureManagementPageID);
```

? 

The event is triggered because you want to open a page. The method exists to open a page, why would it not open a page?

Reply:
also: blindly copying patterns from elsewhere is a surefire way to copy bugs, design mistakes and other issues. There might be good reasons why the code in "Advanced Settings Ext."::OnBeforeOpenCompanySettings() is the way it is but if you don't why then best not to copy it...

Reply:
@grobyns @darjoo I changed the code now, is this what you meant?

Reply:
Yes